### PR TITLE
added login signup

### PR DESCRIPTION
**Pull request by Mandhularajitha**
added login signup

### CODE REVIEW

File: src/features/auth/authSlice.js [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+const initialState = {
+  user: {},
+  token: undefined,
+  status: "idle",
+  error: true,
+};
+
+const saveUserDataInLocalStorage = (userData) => {
+  console.log(userData);
```
Comment (Pratik1005) [inline src/features/auth/authSlice.js:12]: remove log if not needed

File: src/features/auth/authSlice.js [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+};
+
+export const removeUserData = () =>{
+  localStorage.removeItem("userData")
+}
+export const userLogin = createAsyncThunk(
+  "auth/login",
+  async ({username,password}) => {
+    console.log(username);
+    console.log(password);
```
Comment (Pratik1005) [inline src/features/auth/authSlice.js:24]: same here

File: src/features/auth/authSlice.js (hunk 1 above)
Comment (Mandhularajitha) [reply src/features/auth/authSlice.js:12]: ok

File: src/features/auth/authSlice.js (hunk 2 above)
Comment (Mandhularajitha) [reply src/features/auth/authSlice.js:24]: ok

**Review by Pratik1005 — APPROVED** [review]
- There is no deploy preview link or images for better review
- remove commented code if not needed, good to go

**Review by Nikhil-Nirmal — APPROVED** [review]

**Comment by Nikhil-Nirmal** [comment]
Looks good to merge

**Pull request merged**